test/Shared/MsBuildProjectStrings.cs.in

```diff
@@ -555,5 +551,85 @@ namespace Test.Data
     }
 }
 ";
+
+      public const string MSBuildTaskDllPath = @"${OutputPath}\Microsoft.VisualStudio.Web.CodeGeneration.Msbuild.dll";
+      public const string TestCodeGenerationTargetFileText = @"<Project xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
```

Comment:
Does this need to templated? I don't see any substitutions in this value.

Reply:
No substitutions needed for this one. However, as it was needed for test project setup, decided to keep the strings in one file.